recipes/cortexpy/meta.yaml

```diff
@@ -11,7 +11,9 @@ source:
 
 build:
   skip: True  # [py27 or py >= 39]
```

Comment:
we have skipped building for Python 3.8 I think

Reply:
@bgruening What do you mean ? Or rather, what do you suggest ?
The build fails for newer versions of Python ...

Reply:
I see. So, this is the reason why nothing is built.